main.py

```diff
@@ -35,8 +35,8 @@ def __init__(self):
         # Connect threading signals
         self.add_frame.connect(self.setup_videoframe)
         self.fail_add_stream.connect(self.on_fail_add_stream)
-
-        self.vlc_instance = vlc.Instance("--no-xlib")
+        # Opengl performs better on windows, which is odd
```

Comment:
Perhaps we shouldn't have this comment in the source? A more informative comment about the options would be preferred though:
```
self.vlc_instance = vlc.Instance("
    --no-xlib                     # Turn off XInitThreads()
    --vout=opengl            # Force OpenGL as video output module for better performance on Windows
    --avcodec-hw=any     # Hardware decoding
    --avcodec-threads=0 # Number of threads used for decoding, 0 meaning auto
")
```